pkg/reconciler/pipeline/controller.go

```diff
@@ -725,6 +725,15 @@ func (r *pipelineReconciler) updateDesiredState(ctx context.Context, pl *dfv1.Pi
 }
 
 func (r *pipelineReconciler) resumePipeline(ctx context.Context, pl *dfv1.Pipeline) (bool, error) {
+
+	// reset pause timestamp
+	if pl.GetAnnotations()["pauseTimestamp"] != "" {
```

Comment:
Expose the annotation as a constant.

`KeyPauseTimestamp    = "numaflow.numaproj.io/pause-timestamp"`